examples/vcloud-configure-edge/vpn-tunnels.yaml

```diff
@@ -0,0 +1,36 @@
+# Example configuration file for defining VPN tunnels on the vShield Edge Gateway
+#
+# Note that applying this configuration file will replace the current VPN tunnels
+# on the vShield Edge Gateway.
+#
+# Here be dragons:
```

Comment:
I'm not sure that the 'Here be dragons' comment is helpful. Mostly because it's not clear what those dragons are.